modules/node_modules/@ncigdc/components/ProjectsCharts.js

```diff
@@ -87,28 +88,18 @@ const initialState = {
   topGenesSource: [],
 };
 
-function getGenes({ relay, projectIds }: TProps): void {
-  const commonFilters: Array<?TValueFilter> = [
-    significantConsequences,
-    projectIds.length ? {
-      op: 'in',
-      content: {
-        field: 'cases.project.project_id',
-        value: projectIds,
-      },
-    } : null,
-  ].filter(Boolean);
-
+function getGenes({ relay, caseCountFilters }: TProps): void {
   relay.setVariables({
     fetchGeneData: true,
     fmgCaseCount_filters: {
       op: 'AND',
-      content: commonFilters,
+      content: caseCountFilters,
```

Comment:
`caseCountFilters` could be `[]` so probably should check that and make `fmgCaseCount_filters` null if it is.